docker/approved_licenses.json

```diff
@@ -82,6 +82,11 @@
             "url": "https://www.eclipse.org/legal/epl-v10.html",
             "regex": "Eclipse"
         },
+        {
+            "name": "Do What The F*ck You Want To Public License",
```

Comment:
Not clear why we are modifying this file.

Reply:
For this pr it's not needed.
But when I changed the demisto/py3-tools docker image the build [fails](https://app.circleci.com/pipelines/github/demisto/dockerfiles/26582/workflows/54fcc195-5ae8-4ff4-8334-0f9fa80917ca/jobs/32629) on unknown license.
But actually for this pr it's not needed. revert the changes?

Reply:
So please open a separate PR for py3-tools